packages/subgraph/schema.graphql

```diff
@@ -1,13 +1,256 @@
-type Token @entity {
+type TenderizeGlobal @entity { # Global Values/ TODO: better name?
+  "Fixed ID of '1'"
   id: ID!
-  decimals: Int
-  name: String
-  symbol: String
+  "List of congifs for all integrations"
+  configs: [Protocol!]!
+  # TODO: Add global totals 
 }
 
-type Transfer @entity {
+type Protocol @entity {
+  "Name of the protocol eg. 'livepeer'"
   id: ID!
+  "Tenderizer Data"
+  tenderizer: Tenderizer @derivedFrom(field: "protocol")
+  "TenderFarm Data"
+  tenderFarm: TenderFarm @derivedFrom(field: "protocol")
+  "Paritcipating user list"
+  users: [UserProtocol!] @derivedFrom(field: "protocol")
+
+  "Address of the ERC20 token for integration"
+  steak: String!
+  "Address of Tenderizer"
+  tenderizerAddress: String! # Move to Tenderizer?
+  "Address of ERC20 Tender Token"
+  tenderToken: String!
+  "Address of Elastic Supply Pool"
+  esp: String!
+  "Address of liquidity pool for pair"
+  bpool: String!
+  "Address of TenderFarm contract"
+  tenderFarmAddress: String! # Move to TenderFarm?
+}
+
+type Tenderizer @entity {
+  "Name of the protocol eg. 'livepeer'"
+  id: ID!
+  "Protocol entity"
+  protocol: Protocol!
+
+  "Total amount depositied in tenderizer"
+  deposits: BigDecimal!
+  "Total amount withdrawn from tenderizer"
+  withdrawals: BigDecimal!
+  "Total rewards collected from protocol"
+  rewards: BigDecimal!
+  "Total rewards collected from protocol in USD"
+  rewardsUSD: BigDecimal!
+  "Total deposits + total rewards - slashes - total withdrawals"
+  currentPrincipal: BigDecimal!
+  "Current Principle in USD"
+  TVL: BigDecimal!
+
+  "Total amount of protocol fees collected from tenderizer"
+  protocolFees: BigDecimal!
+  "Total amount of protocol fees collected from tenderizer in USD"
+  protocolFeesUSD: BigDecimal!
+  "Total amount of liquidity fees collected from tenderizer"
+  liquidityFees: BigDecimal!
+  "Total amount of liquidity fees collected from tenderizer in USD"
+  liquidityFeesUSD: BigDecimal!
+
+  "Day data for tenderizer"
+  dayData: [TenderizerDay!]! @derivedFrom(field: "tenderizer")
+}
+
+type TenderFarm @entity {
+  "Name of the protocol eg. 'livepeer'"
+  id: ID!
+  "Protocol"
+  protocol: Protocol!
+
+  "Total deposits made into farm"
+  deposits: BigDecimal!
+  "Total withrawals made from farm"
+  withdrawals: BigDecimal!
+  "Total rewards collected from farm"
+  harvest: BigDecimal!
+  "Total rewards collected from farm in USD"
+  harvestUSD: BigDecimal!
+  "Total depsotis - total withdrawals"
+  currentPrincipal: BigDecimal!
+  "Total rewards added"
+  rewards: BigDecimal!
+  "CurrentPrincipal in USD"
+  TVL: BigDecimal!
+
+  "Day data for tenderFarm"
+  dayData: [TenderFarmDay!]! @derivedFrom(field: "tenderFarm")
+}
+
+type TenderizerDay @entity {
+  "<Timestamp rounded to current day by dividing by 86400>_<portocol>"
+  id: ID!
+  "Tenderizer"
+  tenderizer: Tenderizer!
+  "The date beginning at 12:00am UTC"
```

Comment:
Don't need this reverse lookup

Reply:
The main reason if for the derived field in Tenderizer object to auto populate